### PR TITLE
docs: update era-test-node docs with latest release

### PR DESCRIPTION
**Pull request by MexicanAce**
# What :computer: 
* Update In-Memory docs for latest release of `era_test_node` binary

# Why :hand:
* To have up-to-date information in our docs

### CODE REVIEW

File: docs/tools/testing/era-test-node.md [hunk 1]
```diff
@@ -87,48 +77,45 @@ Please note that the existing implementation does not facilitate communication w
...
 
 ```bash
-# era_test_node fork <NETWORK>
-era_test_node fork mainnet
+era_test_node fork testnet
 ```
 
 :::tip
-You can also fork testnet with `era_test_node fork testnet`
+You can also fork testnet with `era_test_node fork mainnet`
```
Comment (dutterbutter) [inline docs/tools/testing/era-test-node.md:87]:
```suggestion
You can also fork mainnet with `era_test_node fork mainnet`
```

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Just that one issue.

File: docs/tools/testing/era-test-node.md (hunk 1 above)
Comment (idea404) [reply docs/tools/testing/era-test-node.md:87]: This line should actually be testnet testnet considering the rest of the file continues reading about testnet

**Review by idea404 — DISMISSED** [review]
Small typo, please see before merging

**Review by idea404 — APPROVED** [review]

**Pull request merged**